examples/js/loaders/ColladaLoader.js

```diff
@@ -1471,11 +1476,24 @@ THREE.ColladaLoader.prototype = {
 							material.emissive.fromArray( parameter.color );
 						break;
 					case 'transparent':
-						// if ( parameter.texture ) material.alphaMap = getTexture( parameter.texture );
+						// if ( parameter.data.texture ) material.alphaMap = getTexture( parameter.data.texture );
 						material.transparent = true;
 						break;
 					case 'transparency':
-						if ( parameter.float !== undefined ) material.opacity = parameter.float;
+						if ( parameter.float !== undefined ) {
+
+							material.opacity = parameter.float;
+
+							if ( parameters[ 'transparent' ] !== undefined ) {
+
+								var opaque = parameters[ 'transparent' ].opaque;
+
+								if ( opaque === 'RGB_ZERO' ) material.opacity = 1 - material.opacity;
+
+							}
+
+						}
+
 						material.transparent = true;
```

Comment:
> I think you should only set material.transparent = true if it is requested, or if material.opacity < 1.

So change this line to:
```javascript
if ( material.opacity < 1 ) material.transparent = true;
```
I guess we can also remove the `material.transparent = true;` statement for the  case `transparent`, see https://github.com/Mugen87/three.js/blob/0e59028607982a876169adfe16fea591faf67ed7/examples/js/loaders/ColladaLoader.js#L1480

Reply:
I would be guessing, so do your best. :)

Reply:
see 2a13b83391b14fc08c047815063bf59c776b66af